### PR TITLE
Fix #4: improve logo image

### LINKED ISSUE #4: Compress the size of the logo image in the header component
## Type of Change
- [ ] New Feature Request
- [ ] Documentation
- [x] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
The logo in the header component is pretty large, and could be optimized further to promote best practices.

## Details
Ideally it would be nice if it was transparent and the header banner could be black, like in CRA.
screen shot 2018-09-22 at 8 07 13 pm

### PR DESCRIPTION
**Pull request by DevLab2425**
## Related Issue

Resolves #4 
## Summary of Changes

- Made logo image transparent
- Refactor header styles to allow for responsive header

Transparent logo on black background:
screen shot 2018-10-24 at 12 23 09 am

### CODE REVIEW

File: src/components/header/header.js [hunk 1]
```diff
@@ -12,9 +12,7 @@ class HeaderComponent extends LitElement {
 
       <header>
 
-        <a href="https://projectevergreen.github.io/">
-          <img src=${logo} alt="Project Evergreen logo"/>
-        </a>
+        <a href="https://projectevergreen.github.io/" style="background-image: url(./${logo})">&nbsp;</a>
```
Comment (thescientist13) [inline src/components/header/header.js]: curious if this is more / less semantic and / or accessible?  👓

Comment (DevLab2425) [reply src/components/header/header.js]: @thescientist13 
I added a `title` attribute to the link to replace the removed `alt` attribute. Not exactly sure what the fallout is though.

**Comment by DevLab2425** [comment]
🤦‍♂️ 
I took the animation out since now the image is the background image. Went for responsiveness over animation. 😏 

The styles are there, but not being applied. I'll check it out.

https://github.com/ProjectEvergreen/create-evergreen-app/pull/19/files#diff-dd96386971ef574fc404d20c28421e0dR23

File: src/components/header/header.js [hunk 2]
```diff
@@ -12,9 +12,7 @@ class HeaderComponent extends LitElement {
 
       <header>
 
-        <a href="https://projectevergreen.github.io/">
-          <img src=${logo} alt="Project Evergreen logo"/>
-        </a>
+        <a href="https://projectevergreen.github.io/" title="Project Evergreen home link" style="background-image: url(./${logo})">&nbsp;</a>
```
Comment (DevLab2425) [inline src/components/header/header.js]: This needs to be `background-image` or it clobbers the `background-*` styles declared in the CSS file.

**Comment by DevLab2425** [comment]
Decided to play with it some more. The animation dropped right back in with no major issue. I tweaked some other header styles to help at different break points. 

I did a terrible job centering the screenshot, but here's a look

screen shot 2018-10-25 at 11 29 01 pm

File: src/components/header/header.js [hunk 3]
```diff
@@ -12,11 +12,11 @@ class HeaderComponent extends LitElement {
...
 
-        <h1>Welcome to Create Evergreen App!</h1>
+        <h1>
+          <small>Welcome to</small>
+          Create&nbsp;Evergreen&nbsp;App!</h1>
```
Comment (DevLab2425) [inline src/components/header/header.js]: Use `&nbsp;` to help against wrapping mid-name.

Comment (thescientist13) [reply src/components/header/header.js]: neat!

File: src/components/header/header.css [hunk 4]
```diff
@@ -12,19 +12,30 @@
 
   & header {
     background-color: #ffffff;
+    display: flex;
```
Comment (thescientist13) [inline src/components/header/header.css]: 💪 

You must be hanging out with that friend of ours 🍝

**Pull request merged**